exercises/leap/cases_test.go

```diff
@@ -1,7 +1,7 @@
 package leap
 
 // Source: exercism/x-common
-// Commit: be6fa53 leap: Rewrite the test cases and their descriptions
+// x-common version: 1.0.0
```

Comment:
An additional thought: I'm just starting a PR to try pulling the canonical-data.json down from Github instead of/prior to doing what it does (I just keep forgetting to update my copy of the x-common repo before regenerating! :man_shrugging: ) and as far as I can see this would mean not being able to include the commit message in the generated file. And if the generator does take the route of being able to get the current json data via http as well as from an offline repo I assume we wouldn't want to sometimes have the commit message, sometimes not?